strings/src/main/res/values/strings.xml

```diff
@@ -1185,6 +1185,9 @@
     <!-- Label for entity that has been created offline and is not on the server yet -->
     <string name="offline">Offline</string>
 
+    <!-- Error message displayed when a user opens a form with an unrecognized entity version -->
+    <string name="unrecognized_entity_version">The form contains an unrecognized entity version. Please update the form and download it again.</string>
```

Comment:
@alyblenkin @lognaturel do you think that message should say something more? Any recommendations?

Reply:
I'm wondering if data collectors will be familiar with Entities? And I liked the part @lognaturel added in the story about contacting their manager or someone that can help.

> **Unrecognized Entity version** 
The form contains unrecognized information. Update the form and download it again. If you keep having this problem, report it to the person who asked you to collect data.

Reply:
Do we want to use the specific error title 'Unrecognized Entity Version'? Currently, we display errors under a generic 'Error Occurred' title, but we may want to refine this, especially now that broken forms won't be removed immediately after download.

Reply:
Message I proposed:

> This form is not supported by this version of ODK Collect. Please upgrade Collect. If you keep having this problem, report it to the person who asked you to collect data.

> Form Entities spec version: ${version}

I think in general the action will be to update Collect so we should highlight that. I think we should also include the version for troubleshooting purposes.

I think it's ok to use a generic error title for the modal. We could go for something stronger like "Form can't be used"

Reply:
I was going back and forth on the title. Using Entities in the title still isn't great because the data collector won't know what that means. Trying to think of ways not to use "Entity"by saying "linked" or "shared between forms"

Here are two more ideas:

> **Error occurred** 
> Your form contains unrecognized information linked with other forms. Update the form and download it again. If you keep having this problem, report it to the person who asked you to collect data.

> **Error occurred** 
Your form contains an unrecognized data. To ensure you have the right version, update the form and download it again. If the issue persists, contact the person who requested the data collection.

Reply:
In this case, there isn't a form design fix. We want the user to upgrade Collect.

Reply:
Dismiss what I just shared - @lognaturel's comment didn't show up when I started thinking about alternatives!

Reply:
Ohh, GitHub is getting us all confused! Definitely happy to play with the wording but I think that’s the content we need.

Reply:
yes,,,there are weird delays happening! I like your proposal a lot and we can always iterate later.